src/app/sb-admin/components/pages/sb-user/sbuser.component.html

```diff
@@ -63,12 +63,26 @@ <h5 class="m-0">{{ 'COMMON_USER' | i18next}}</h5><br>
                         <td>
                             <div class="flex">
                                 <button [disabled]="!user.status" pButton pRipple icon="pi pi-pencil" class="p-button-rounded p-button-info mr-2" (click)="editRole(user)"></button>
-                                <button pButton pRipple icon="pi pi-trash" class="p-button-rounded p-button-warning"></button>
+                                <button pButton pRipple icon="pi pi-trash" class="p-button-rounded p-button-warning" (click)="blockUnblockUser(user)" *ngIf="user.status"></button>
+                                <button pButton pRipple icon="pi pi-unlock" class="p-button-rounded p-button-warning" (click)="blockUnblockUser(user)" *ngIf="!user.status"></button>
                             </div>
                         </td>
                     </tr>
                 </ng-template>
             </p-table>
+
+            <p-dialog [(visible)]="blockUnblockUserDialog" header="Confirm" [modal]="true" >
+                <div class="flex align-items-center justify-content-center">
+                    <i class="pi pi-exclamation-triangle mr-3" style="font-size: 2rem"></i>
+                    <span *ngIf="user?.status === 1 ">{{'USER_BLOCK_CONFIRMATION' | i18next}} <b>{{user?.firstName}}</b>?</span>
+                    <span *ngIf="user?.status === 0" >{{'USER_UNBLOCK_CONFIRMATION' | i18next}} <b>{{user?.firstName}}</b>?</span>
+                </div>
+                <ng-template pTemplate="footer">
+                    <button pButton pRipple icon="pi pi-times" class="p-button-text" label="{{'COMMON_NO' | i18next}}" (click)="blockUnblockUserDialog = false"></button>
```

Comment:
Use only one button tag

Reply:
we are showing two button on a single dialog at a time.